### DISCUSSION TITLE
Toggling visibility of list of blocked websites

### DISCUSSION THREAD

**Discussion by aryamm20** [SelfControlApp/selfcontrol#728] [Q&A] [answered]
Is there a way to toggle the preview of the websites in one's blocklist? Perhaps so it says only the number of websites in one's blocklists but not which ones they are?

<img width="611" alt="Screen Shot 2021-10-08 at 5 48 17 AM" src="https://user-images.githubusercontent.com/74794890/136559656-1dbc40c8-b95b-46d5-9b5a-d4b0a901ba2c.png">

**Comment by cstigler** [comment, marked as answer]
This is a great question - there isn't, but I can totally see why you'd want that! We should make it toggleable in a future version - I'll convert this into an issue to help me remember to put that into the next release. But don't hold your breath, it'll probably be a while before that happens :)

For now as a workaround, just edit your blocklist and make the first site in your list a fake domain that's very long, like `abcdefghijklmnoprstuvwxyzabcdefghijklmnoprstuvwxyz.com.com`. That'll make it only show the number, since it won't have the space to show the domains.

**Reply by cstigler** [reply]
The issue to follow for this feature: https://github.com/SelfControlApp/selfcontrol/issues/730

**Reply by aryamm20** [reply]
Got it, thanks!
